bionemo-recipes/recipes/codonfm_native_te/README.md

```
@@ -0,0 +1,198 @@
+[`bionemo-recipes/models/codonfm/`](../../models/codonfm/). Pre-trained checkpoints for the "exact" TE architecture are
+available on the Hugging Face Hub (see the [codonfm_ptl_te README](../codonfm_ptl_te/README.md#pre-trained-models) for
+links). A native_te checkpoint trained with this recipe will be uploaded in the future.
+
+## Performance Benchmarks
+
+Under development. For TE acceleration benchmarks comparing against the original Xformers-based implementation, see the
+[codonfm_ptl_te benchmarks](../codonfm_ptl_te/README.md#nvidia-transformerengine-optimization-benchmarks).
+
+## Repository Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)